recordedit/recordEdit.app.js

```diff
@@ -1,6 +1,27 @@
 (function() {
     'use strict';
+/* Configuration of the Recordedit App */
+    angular.module('chaise.configure-recordedit', [
+        'chaise.modal',
+        'chaise.utils',
+        'ermrestjs',
+        'ngCookies',
+        'ui.bootstrap'
+    ])
+
+    .run(['ERMrest', function (ERMrest) {
+        try {
+            ERMrest.onload().then(function () {
+                angular.element(document).ready(function(){
+                    angular.bootstrap(document.getElementById("recordedit"), ["chaise.recordEdit"]);
+                });
+            });
+        } catch (exception) {
+            throw exception;
```

Comment:
Which part of this might throw an error?

Reply:
I'm not sure. I wasn't sure if onload would error out. I was thinking it could return an error/exception promise that we weren't handling.

Reply:
It won't throw any explicit errors. And even if it does, having a try-catch that just throws the error seems completely pointless. You could add a reject callback that throws the exception. But that wouldn't be necessary either since the `ERMrest.onload` won't reject the callback.